unzip_secrets.sh

```diff
@@ -0,0 +1 @@
+unzip qvain_ops_sec.zip
```

Comment:
this is not a valid shell script. Missing the https://en.wikipedia.org/wiki/Shebang_(Unix)